### PR TITLE
Added Badges

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added Badges

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

**Comment by huanghua78** [comment]
Let me ask @mehjoshi to review this.

**Comment by Saumya-Sunder** [comment]
Sure @huanghua78 please refer to issue https://github.com/Seagate/cortx-motr/issues/735

**Review by huanghua78 — APPROVED** [review]

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: README.rst [hunk 1]
```diff
@@ -3,23 +3,36 @@
...
 
-.. Slack Badge  - https://img.shields.io/badge/chat-on%20Slack-blu
-
-.. image:: https://img.shields.io/badge/chat-on%20Slack-blue
-        :target: https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?
-        :alt: Slack chat badge
+.. License Badge - https://img.shields.io/badge/License-Apache%202.0-blue.svg
 
+.. image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+        :target: https://github.com/pujamudaliar/cortx-motr/blob/main/LICENCE
```
Comment (madhavemuri) [inline README.rst:9]: Change https://github.com/pujamudaliar/cortx-motr/blob/main/LICENCE

to https://github.com/Seagate/cortx-motr/blob/main/LICENCE

Comment (huanghua78) [reply README.rst:9]: yes. I did not notice this.

File: README.rst [hunk 2]
```diff
@@ -3,23 +3,36 @@
...
 
 .. image:: https://api.codacy.com/project/badge/Grade/e047436e66e54d67b911294ad7fe8b4a
         :target: https://app.codacy.com/gh/Seagate/cortx-motr?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-motr&utm_campaign=Badge_Grade
          :alt: Codacy Badge 
         
-.. License Badge - https://img.shields.io/badge/License-Apache%202.0-blue.svg
+.. Slack Badge  - https://img.shields.io/badge/chat-on%20Slack-blu
+
+.. image:: https://img.shields.io/badge/chat-on%20Slack-blue
+        :target: https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?
```
Comment (huanghua78) [inline README.rst:21]: Hello @Saumya-Sunder , I see  the top level cortx repo uses this URL for slack.
Should we use it too?
```
https://cortx.link/join-slack
```

File: README.rst (hunk 1 above)
Comment (Saumya-Sunder) [reply README.rst]: Fixed already

File: README.rst (hunk 2 above)
Comment (Saumya-Sunder) [reply README.rst:21]: Sure, that is more up to date than these READMEs. Thanks @huanghua78

**Review by huanghua78 — APPROVED** [review]

**Pull request merged**